apm-checkstyle/checkStyle.xml

```diff
@@ -110,4 +110,11 @@
             <property name="path" value="apm-sniffer/(apm-sdk-plugin|optional-plugins)/.+/src/main/.+Instrumentation.java$" />
         </module>
     </module>
+
+    <module name="RegexpHeader">
```

Comment:
We're using `RegexpHeader` checker module here because the existed license headers vary, some of them have more spaces between sentences or have more blank lines, which I don't think is a problem, so I just use `RegexpHeader` and ignore blank lines or spaces, like line 16 @wu-sheng

Reply:
This is not a problem. I am just asking about your only check `.java`.